sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -34,7 +34,8 @@
 from sklearn.utils._testing import assert_array_almost_equal
 from sklearn.utils._testing import assert_array_equal
 from sklearn.utils._testing import ignore_warnings
-
+from unittest.mock import Mock
```

Comment:
Nit: Can you move this import to line 9 below `import warning`? This way the "first party Python modules" are at the top of the file.

Reply:
Sure! I will do that immediately